gitindex/index.go

```diff
@@ -533,6 +551,7 @@ func indexGitRepo(opts Options, config gitIndexConfig) error {
 				Name:              key.FullPath(),
 				Content:           contents,
 				Branches:          brs,
+				Scores:            score(key.FullPath()),
```

Comment:
welcome to the world of micro optimizations, sorry. FullPath does work and allocates. Maybe call FullPath once?

Reply:
good catch. I love micro optimizations.